config/opa-service/policy.rego

```diff
@@ -56,7 +56,13 @@ default user_permitted = false
 # Check for token validity
 is_token_valid {
   now := time.now_ns() / 1000000000
-  token.payload.nbf <= now
+  token.payload.exp >= now
+}
```

Comment:
let's update the readme accordingly:
https://github.com/orchestracities/opa-authz/blob/gabriele-dev/README.md?plain=1#L200